server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/ActiveRuleDao.java

```diff
@@ -140,6 +136,10 @@ public ActiveRuleParamDto selectParamByKeyAndName(ActiveRuleKey key, String name
     return null;
   }
 
+  /**
+   * @deprecated currently used only by tests
+   */
+  @Deprecated
   public List<ActiveRuleParamDto> selectAllParams(DbSession dbSession) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)